plugins/pencilblue/controllers/section.js

```diff
@@ -39,32 +39,32 @@ module.exports = function(pb) {
         var init = function(err) {
             //get content settings
             var serviceContext = self.getServiceContext();
-            var contentService = new pb.ContentService({site: self.site, onlyThisSite: serviceContext.onlyThisSite});
+            var contentService = new pb.ContentService(context);
             contentService.getSettings(function(err, contentSettings) {
                 if (util.isError(err)) {
                     return cb(err);
                 }
                 //create the service
                 self.contentSettings = contentSettings;
-                var asContext = self.getServiceContext();
+                var asContext = serviceContext;
```

Comment:
`asContext` was actually doing that already and I changed it to this. `serviceContext` was an unused variable prior, so I took advantage of it rather than deleting it. However, the change above will be making use of it again so I'll change this back for the same reasoning as above.